modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/util/JournalConverterImpl.java

```diff
@@ -181,13 +190,24 @@ private void _addDDMFields(
 		for (Element dynamicElementElement : dynamicElementElements) {
 			if (!ddmFormField.isTransient()) {
 				Field ddmField = _getField(
```

Comment:
I would declare this variable following its usage (in `if (existingDDMField != null) {`)